src/components/EditCoursePage/EditCourseForm.jsx

```diff
@@ -300,6 +303,12 @@ export class BaseEditCourseForm extends React.Component {
     subjectOptions.unshift({ label: '--', value: '' });
     programOptions.unshift({ label: '--', value: '' });
 
+    const toggleDirectionChange = () => {
+      this.setState(prevState => ({
```

Comment:
I think we need to take care of the other fields in the state too? So something like `{...prevState, direction: newDirection}` ?